GOVERNANCE.md

```diff
@@ -1,114 +1,295 @@
+
+The Operator Framework community will work on developing best practices,
+fostering collaboration, and working on improving tool interoperability.
+
+## Individual Subproject Governance
+
+All active Approvers of each subproject, as defined in the Contributor
+Ladder, are members of that project's Approvers Committee, which governs
+that project. The subproject's Approvers Committee is responsible for
+the following project governance activities:
+
+* Ensuring that the subproject creates and publishes regular releases;
+* Holding regular and open subproject-wide discussions on feature planning and issues;
+* Regular review of project contributors for advancement on the Contributor Ladder;
+* Following subproject governance and ensuring that the subproject
+    functions according to the Operator Framework values.
+* Making final decisions on subproject changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their subproject and referring violations
+  to the Code of Conduct Committee.
+
+Additionally, the project's Approvers Committee will select, by majority
+vote, one representative to the Steering Committee. This representative
+need not be a member of the subproject's Approvers Committee, and will
+be replaced or renewed by the committee annually.
+
+Should a member of the Approvers Committee cease being active in the subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Committee members, or a
+majority vote of the Steering Committee.
+
+### Default Subproject Governance
+
+Subprojects without governing documents default to:
+
+#### Subproject Approvers Committee Responsibilities
+
+All active Approvers of the project, as defined in the Operator
+Framework Contributor Ladder, are members of the Maintainer Committee,
+which governs that project. The project's Approver Committee is
+responsible for the following project governance activities:
+
+* Ensuring that the project creates and publishes regular releases;
+* Holding regular, project-wide discussions on issues and planning for the project;
+* Monthly review of project contributors for advancement on the Contributor Ladder;
+* Making final decisions on project changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their project and referring violations to the Code of Conduct Committee.
+
+Should a member of the Approver Committee cease being active in the
+project, violate the Code Of Conduct, or need to be removed for some
+other reason, they may be removed by a 2/3 majority vote of the other
+Committee members, or a vote of the Steering Committee.
+
+#### Subproject Steering Committee Representative
+
+Additionally, the Approver Committee will select, by an election
```

Comment:
Add a blurb here explicitly saying if the subproject only puts forward a single nominee, then don't bother holding an election.